ui/components/gridTable/gridTable.scss

```diff
@@ -84,10 +84,10 @@ $grid-column-gap: 2px;
 
   .epic-table-header-grid {
     .skillTP-grid {
-      color: darken($tp-color, 10%);
+      color: lighten($tp-color, 25%);
     }
     .skillCP-grid {
-      color: darken($cp-color, 10%);
+      color: darken($cp-color, 20%);
```

Comment:
The header CP and TP colors were a bit harder to see.
Now:
![image](https://user-images.githubusercontent.com/5629800/209596377-8abfd667-907f-4be3-b659-64f3a756e822.png)